popup/popup.js

```diff
@@ -228,19 +238,32 @@ function showStyles(styles) {
     return;
   }
 
-  const enabledFirst = prefs.get('popup.enabledFirst');
-  styles.sort((a, b) => (
-    enabledFirst && a.enabled !== b.enabled
-      ? !(a.enabled < b.enabled) ? -1 : 1
-      : a.name.localeCompare(b.name)
-  ));
-
   const container = document.createDocumentFragment();
-  styles.forEach(style => createStyleElement({style, container}));
+  sortStyles({
```

Comment:
I think we can change the signature of `createStyleElement` to `createStyleElement(styleMeta) -> Element` and define a compare function, so we can use it like:
```js
const elements = styles.map(createStyleElement);
if (pref.get('popup.enabledFirst')) {
	elements.sort(compareEnabled);
}
elements.forEach(e => container.appendChild(e));
```
Therefore we can avoid using a property getter.

Reply:
Seems like this whole addition isn't as popular as I supposed... I'll hold off on this change while we wait for more feedback.

Reply:
> I'm not sure what changes need to be made.

Maybe I can push the change to this branch?

> I'm inclined to forget about it and just close this PR.

I think we can add this option since you and @narcolepticinsomniac think this feature is useful. Not enabled by default though.

Reply:
> Maybe I can push the change to this branch?

Please go ahead. I'd appreciate the help.